features/sct-section-header.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Header stuff
+  Scenario: Basic
+    Given a section
+     Then section.header is a Header object
+
+    Given a header having a definition
+     Then header.is_linked_to_previous is False
+
+    Given a header having a definition
+     Then header.text is the text contained in the header
```

Comment:
Sounds good.

Makes sense about grouping all the `Given an x...` into a separate file starting with the 3 letter abbreviation for x.

wip seems handy. sounds good!